managed/models/database.go

```diff
@@ -820,15 +825,33 @@ var databaseSchema = [][]string{
 // aleksi: Go's zero values and non-zero default values in database do play nicely together in INSERTs and UPDATEs.
 
 // OpenDB returns configured connection pool for PostgreSQL.
-func OpenDB(address, name, username, password string) (*sql.DB, error) {
+// OpenDB just validate its arguments without creating a connection to the database.
+func OpenDB(params SetupDBParams) (*sql.DB, error) {
 	q := make(url.Values)
-	q.Set("sslmode", "disable")
+	addSSL := true
+	switch params.SSLMode {
+	case "require":
+		q.Set("sslmode", "require")
+	case "verify-ca":
+		q.Set("sslmode", "verify-ca")
+	case "verify-full":
+		q.Set("sslmode", "verify-full")
+	default:
+		q.Set("sslmode", "disable")
+		addSSL = false
+	}
+
+	if addSSL {
+		q.Set("sslrootcert", params.SSLCAPath)
+		q.Set("sslcert", params.SSLCertPath)
+		q.Set("sslkey", params.SSLKeyPath)
+	}
```

Comment:
```suggestion
	```suggestion
        sslMode = params.SSLMode
        if sslMode == "" {
            sslMode = "disable"
        }
        q.Set("sslmode", sslMode)

	if sslMode != "disable" {
		q.Set("sslrootcert", params.SSLCAPath)
		q.Set("sslcert", params.SSLCertPath)
		q.Set("sslkey", params.SSLKeyPath)
	}
```